packages/amplify_api/android/src/main/kotlin/com/amazonaws/amplify/amplify_api/FlutterGraphQLApiModule.kt

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amazonaws.amplify.amplify_api
+
+import android.os.Handler
+import android.os.Looper
+import com.amazonaws.amplify.amplify_api.types.FlutterApiErrorMessage
+import com.amplifyframework.api.aws.GsonVariablesSerializer
+import com.amplifyframework.api.graphql.SimpleGraphQLRequest
+import com.amplifyframework.core.Amplify
+import io.flutter.plugin.common.MethodChannel
+
+class FlutterGraphQLApiModule {
+    companion object {
```

Comment:
Seems to be a pattern in this PR if using static classes to warehouse some code.

In this one, look at the signatures of the methods. They seem to all need a method channel. Maybe you should be passing that in as a dependency, and creating an instance of this class.

What does `Module` mean?

Alternately, why don't you split this out into two things, a `FlutterGraphQlQuery` and a `FlutterGraphQlMutation`. You can make instances of them, and passa method channel as a dependency. They'll have better names.

They're looking like wrappers around the native versions of those calls, right?